From 64be190a1c42a901ff5dc31bb1e9e5959e063555 Mon Sep 17 00:00:00 2001
From: LongLuu <long.luu.ur@renesas.com>
Date: Fri, 26 Mar 2021 14:17:27 +0700
Subject: [PATCH 148/151] arm64: dts: renesas: r9a07g044l2-smarc: Enable SCIF2
 node.

This commit enable SCIF2 node for RZ/G2L Pro-board

Signed-off-by: LongLuu <long.luu.ur@renesas.com>
---
 arch/arm64/boot/dts/renesas/r9a07g044l2-smarc.dts | 13 +++++++++++++
 1 file changed, 13 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r9a07g044l2-smarc.dts b/arch/arm64/boot/dts/renesas/r9a07g044l2-smarc.dts
index b53be2d..419c463 100644
--- a/arch/arm64/boot/dts/renesas/r9a07g044l2-smarc.dts
+++ b/arch/arm64/boot/dts/renesas/r9a07g044l2-smarc.dts
@@ -36,6 +36,7 @@
 		i2c1 = &i2c1;
 		i2c3 = &i2c3;
 		serial0 = &scif0;
+		serial2 = &scif2;
 		ethernet0 = &eth0;
 		ethernet1 = &eth1;
 	};
@@ -402,6 +403,11 @@
 		function = "scif0";
 	};
 
+	scif2_pins: scif2 {
+		groups = "scif2_data_e", "scif2_ctrl_e" ;
+		function = "scif2";
+	};
+
 	sd0_pwr_en {
 		gpio-hog;
 		gpios = <RZG2L_GPIO(4, 1) GPIO_ACTIVE_HIGH>;
@@ -531,6 +537,13 @@
 	status = "okay";
 };
 
+&scif2 {
+	pinctrl-0 = <&scif2_pins>;
+	pinctrl-names = "default";
+	uart-has-rtscts;
+	status = "okay";
+};
+
 /*
  * SW1[2] should be at position 1/HIGH to enable uSD card CN3
  * un-comment the below node and comment out eMMC node.
-- 
2.7.4

